src/example/example.service.ts

```diff
@@ -0,0 +1,93 @@
+export class ExampleService {
+  public calculateAverage(numbers: number[]): number {
+    if (!numbers.length) return 0;
+    let sum = 0;
+    for (const num of numbers) {
+      sum += num;
+    }
+    return sum / numbers.length;
+  }
```

Comment:
PullProofAI

The `calculateAverage` method in the `ExampleService` class calculates the average of numbers in an array. It would be beneficial to handle the case where the input array is empty to avoid division by zero. Adding a check for an empty array before performing the division operation is recommended.

```diff
+ if (!numbers.length) return 0;
```



<!-- This is an auto-generated comment by PullProofAI -->